client-rpc/Cargo.toml

```diff
@@ -25,8 +25,8 @@ secstr = { version = "0.3.2", features = ["serde"] }
 websocket = "0.23.0"
 futures = "0.1.28"
 tokio = "0.1.22"
```

Comment:
There are still these unused dependencies in `client-rpc`. For example, `tokio`, `websocket`, etc. Please remove them.

Reply:
oh, i forgot to check,
i will remove those.

Reply:
also i will check CI issue